src/coreclr/jit/emitxarch.cpp

```diff
@@ -9918,6 +9924,11 @@ void emitter::emitDispIns(
             {
                 printf("L_M%03u_" FMT_BB, emitComp->compMethodID, id->idAddr()->iiaBBlabel->bbNum);
             }
+
+            if (((instrDescJmp*)id)->idjIsJmpAlways)
+            {
+                printf(" ; BB_ALWAYS");
```

Comment:
```suggestion
                printf(" ; BBJ_ALWAYS");
```

This doesn't make sense to me to print `BBJ_ALWAYS` -- we know that if a `jmp` is emitted, we've generated code for a `BBJ_ALWAYS`, right?

Reply:
I've removed this and included something in the `emitJumpList()` that shows more clearly what is going on. the jump list wasn't showing size, was always showing 0's as the offset and didn't have all the jump information needed to understand what alterations have been made. Again, if we change the name from `isJmpAlways' i'll update the annotation to make it clear what it's supposed to be telling you.